xbmc/music/MusicDatabase.cpp

```diff
@@ -3689,6 +3698,8 @@ bool CMusicDatabase::GetSongsNav(const std::string& strBaseDir, CFileItemList& i
 
 void CMusicDatabase::UpdateTables(int version)
 {
+  CLog::Log(LOGINFO, "%s - updating tables", __FUNCTION__);
```

Comment:
I left that in for a reason. It's kinda hard to see when the UpdateTables happen.
```
    try
    {
      // drop old analytics, update table(s), recreate analytics, update version
      m_pDB->drop_analytics();
      UpdateTables(version);
      CreateAnalytics();
      UpdateVersionNumber();
    }
```
Each of these methods logs something when they start, so the UpdateTables is kinda lost